src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -21,6 +21,7 @@ import { getI18N } from "../../../utilities/i18n-utilties";
 import { SPACE_SM } from "../../../styles/spacing";
 import { navigate } from "@reach/router";
 import { green6, grey6, red6 } from "../../../styles/colors";
+import { ContentLoading } from "../../../components/loader/ContentLoading";
 
```

Comment:
In this file you also make"

```jsx
 <Icon type="check-circle"  style={{ marginRight: SPACE_SM, color: green6 }}  />
 <Icon type="close-circle" style={{ marginRight: SPACE_SM, color: red6 }} />
<Icon type="loading" style={{ marginRight: SPACE_SM, color: grey6 }} />
```
Can you make these standard components

Reply:
Done in 6de64b4b1c59f8f50d554093d2ced426d7b6d605